datajoint/condition.py

```diff
@@ -8,8 +8,29 @@
 import decimal
 import numpy
 import pandas
+import json
 from .errors import DataJointError
 
+JSON_PATTERN = re.compile(
+    r"^(?P<attr>\w+)(\.(?P<path>[\w.*\[\]]+))?(:(?P<type>[\w(,\s)]+))?$"
+)
+
+
+def translate_attribute(key):
+    match = JSON_PATTERN.match(key)
+    if match is None:
+        return match, key
+    match = match.groupdict()
+    if match["path"] is None:
+        return match, match["attr"]
+    else:
+        return match, "json_value(`{}`, _utf8mb4'$.{}'{})".format(
+            *{
+                k: ((f" returning {v}" if k == "type" else v) if v else "")
+                for k, v in match.items()
+            }.values()
```

Comment:
Here it's not necessary to create the dict for taking its `values()`
```suggestion
            *[
               ((f" returning {v}" if k == "type" else v) if v else "")
                for k, v in match.items()
            ]
```